guacamole-common-js/src/main/webapp/modules/Tunnel.js

```diff
@@ -214,6 +219,29 @@ Guacamole.HTTPTunnel = function(tunnelURL, crossDomain) {
      */
     var receive_timeout = null;
 
+    /**
+     * Additional headers to be sent in tunnel requests. This dictionary can be
+     * populated with key/value header pairs to pass information such as authentication
+     * tokens, etc.
+     *
+     * @private
+     */
+    var extraHeaders = extraTunnelHeaders || {}
+
+    /**
+     * Adds the configured additional headerss to the given request.
+     *
```

Comment:
`request` is not an optional parameter of this function, and thus must not be within brackets here.